travis-status.ts

```diff
@@ -209,9 +209,9 @@ export default class TravisStatusIndicator {
 		}
 	}
 
-	private getTeravis(): any {
+	private getTravis(): any {
 		if (this._travis == null) {
-			this._travis = Travis.new({
+			this._travis = new Travis({
```

Comment:
@felixrieseberg This was the main issue. `Travis.new` was not working. Needed to be `new Travis`